src/main/java/appeng/items/tools/NetworkToolItem.java

```diff
@@ -127,34 +119,17 @@ public boolean serverSideToolLogic(UseOnContext useContext) {
         // The network tool has special behavior for machines hosting world-accessible nodes
         var nodeHost = GridHelper.getNodeHost(level, pos);
 
-        var bs = level.getBlockState(pos);
-        if (!InteractionUtil.isInAlternateUseMode(p)) {
-            if (nodeHost == null && bs.rotate(Rotation.CLOCKWISE_90) != bs) {
-                bs.neighborChanged(level, pos, Blocks.AIR, pos, false);
-                p.swing(hand);
-                return !level.isClientSide;
-            }
+        if (p.containerMenu instanceof AEBaseMenu) {
+            return true;
         }
```

Comment:
Good point, I think previously it was to prevent "double" opening the menu when it was already opened (due to the packet arriving late/twice?)
I'll get rid of this and make the method private since its no longer called by the ClickPacket handler.